src/test/java/com/julienviet/pgclient/PgDataTypeBase.java

```diff
@@ -0,0 +1,5 @@
+package com.julienviet.pgclient;
+
+public class PgDataTypeBase extends PgConnectionTest {
```

Comment:
make it abstract and name it `DataTypeTestBase`

Reply:
also it should not extend PgConnectionTest because it will run the PgConnectionTest and we only want to run the data type tests